interop-testing/src/main/java/io/grpc/testing/integration/XdsFederationTestClient.java

```diff
@@ -76,6 +76,7 @@ public void run() {
   private int soakMinTimeMsBetweenRpcs = 0;
   private int soakRequestSize = 271828;
   private int soakResponseSize = 314159;
+  private int numThreads = 1;
```

Comment:
nit: we don't need to make numThreads configurable for `XdsFederationTestClient`, we can leave it hardcoded at 1.

I.e. we can remove the new flag here